mm2src/adex_cli/src/adex_app.rs

```diff
@@ -11,9 +10,9 @@ pub(super) struct AdexApp {
 }
 
 impl AdexApp {
-    pub(super) fn new() -> Result<AdexApp> {
-        let config = AdexConfigImpl::read_config()?;
-        Ok(AdexApp { config })
+    pub(super) fn new() -> AdexApp {
+        let config = AdexConfigImpl::read_config().unwrap_or_default();
```

Comment:
Should `read_config` return `AdexConfigImpl` straight away instead of result? right now these warning https://github.com/KomodoPlatform/atomicDEX-API/blob/c4d6c71dff4c175621f7d47813727b8d3dec5bee/mm2src/adex_cli/src/adex_config.rs#L101-L106 will not be shown if `read_config` returned an error

Reply:
Another comment is I would like to make methods as they are supposed to behave and make decisions on business level.
In this certain case `AdexConfigImpl::read_config()` is anticipated to return result if something has happened and on the Application level it's decided to use the `default` one.
Thank you again )

Reply:
Thanks for the detailed explanation, I agree that the current approach is fine.